PWGCF/FemtoWorld/TableProducer/femtoWorldProducerTask.cxx

```diff
@@ -269,7 +269,8 @@ struct femtoWorldProducerTask {
     ccdb->setCaching(true);
     ccdb->setLocalObjectValidityChecking();
 
-    long now = std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::system_clock::now().time_since_epoch()).count();
+    // changed long to float because of the MegaLinter
+    float now = std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::system_clock::now().time_since_epoch()).count();
```

Comment:
time is an integral! Better to write int32_t or uint32_t or uint64_t as MegaLinter advises